pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -57,8 +57,12 @@ let
     hasAttr getAttr optional optionals optionalString optionalAttrs maintainers platforms;
 
   # Dependencies that are required to build kernel modules
-  moduleBuildDependencies = [ perl libelf ]
-    ++ optional (lib.versionAtLeast version "5.13") zstd;
+  moduleBuildDependencies = [
```

Comment:
Are these nativeBuildDependencies or buildDependencies?

Reply:
perl and deterministic-uname are nativeBuildInputs while libelf is a buildInputs, zstd contains binaries and libraries so it could be a nativeBuildInputs or a buildInputs depending on why it was added

Reply:
I don't think it makes sense to put them in nativeBuildInputs then.

Ideally, they should be split properly.

Reply:
yep but thats the way it was before this pr

Reply:
You're merging them in a bunch of places though?

Reply:
just making them consistent